indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -73,15 +73,32 @@
                     </ul>
                 </div>
                 <div class="group">
-                    <a class="i-button icon-checkmark js-requires-selected-row js-dialog-action accept">
-                        {% trans %}Accept{% endtrans %}
+                    <a class="i-button icon-checkmark accept js-requires-selected-row"
+                       data-href="{{ url_for('.manage_judge_abstracts', event) }}"
+                       data-title="{% trans %}Accept selected abstracts{% endtrans %}"
+                       data-params-selector="#abstract-list input[name=abstract_id]:checked"
+                       data-params='{"action": "{{ actions.accept.name }}"}'
```

Comment:
But it's **extremely** unlikely that those enum entries change (especially just their values). and generating JSON using `{"foo" "{{ bar }}"}` is pretty dirty (would fail with any special chars, even though that's not going to happen here) 
And `{{ {'foo': bar}|tojson|forceescape }}` would be even uglier..